README.md

```diff
@@ -494,6 +494,26 @@ foo = {one: one, two}
   s"""foo bar"""
   f"""foo bar"""
   ```
+  
+### Git Rebase
+
+* Git Rebase Commands
+  ```
+  pick -> fixup
+  p -> fixup
+  fixup -> reword
+  f -> reword
+  reword -> edit
+  r -> edit
+  edit -> squash
+  e -> squash
+  squash -> exec
+  s -> exec
+  exec -> drop
+  x -> drop
+  drop -> pick
+  d -> pick
+  ```
```

Comment:
This is actually easier to read in the code than in the documentation :D. I wonder if we can make it shorter. Maybe, similar to the implementation, we could write `pick -> fixup -> reword -> ... -> pick`, and then have the single-line versions each on a line, `p -> fixup`, `f -> reword`, and so on. Maybe that would make it slightly more readable?

Also, would it be okay if you added the same documentation in the help docs as well? Somewhere around here, I guess: https://github.com/AndrewRadev/switch.vim/blob/f9826833a84d600811d4214059e30e2892c24dbf/doc/switch.txt#L568. (I should really write a script that generates one from the other...)